src/example/p4p5/setuphandlers.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from Products.CMFPlone import interfaces as Plone
+from Products.CMFQuickInstallerTool import interfaces as QuickInstaller
+from zope.interface import implementer
+
+
+@implementer(Plone.INonInstallable)
+class HiddenProfiles(object):
+
+    def getNonInstallableProfiles(self):
+        """Do not show on Plone's list of installable profiles."""
+        return [
+            u'example.p4p5:base',
+            u'example.p4p5:uninstall',
+            u'example.p4p5:uninstall-base',
+        ]
+
+
+@implementer(QuickInstaller.INonInstallable)
+class HiddenProducts(object):
```

Comment:
is it needed? (is HiddenProducts mandatory if we want just to use HiddenProfiles?)